core-services/prow/02_config/_plugins.yaml

```diff
@@ -2593,6 +2597,9 @@ lgtm:
   - ansible-collections/community.okd
   - openshift-priv/community.okd
   review_acts_as_lgtm: true
+- repos:
+  - openshift/route-monitor-operator
+  review_acts_as_lgtm: true
```

Comment:
Why is this here?

Reply:
Is it a bad idea?
if someone clicks the lgtm button in the review, the label gets added, or not?